### PR TITLE
NEW: load settings using env file

### LINKED ISSUE #7: load settings using env file
we need to be able to load django setting from an env file. this feature will help us to manage settings using Kubernetes configmaps.

### PR DESCRIPTION
**Pull request by peymanslh**
Changes:
- Add `.sample_env` file
- Load `python-dotenv` package in django `manage.py` file
- Add `python-dotenv` package in `requirements.txt`

closes: #7 

P.S. `pyproject.toml` file is edited by pre-commit rules

### CODE REVIEW

**Comment by hramezani** [comment]
Do we need to copy `.sample_env` to `.env` to use it? If so, please add documentation for it

**Comment by peymanslh** [comment]
> Do we need to copy `.sample_env` to `.env` to use it? If so, please add documentation for it

Yes, and I updated the [README](https://github.com/mirzakhany/django_boilerplate/pull/19/commits/72492b4ea828ac31a561b53ee6007b11c2057c11) file. Is that ok, or I should add more description?

**Comment by hramezani** [comment]
> > Do we need to copy `.sample_env` to `.env` to use it? If so, please add documentation for it
> 
> Yes, and I updated the [README](https://github.com/mirzakhany/django_boilerplate/pull/19/commits/72492b4ea828ac31a561b53ee6007b11c2057c11) file. Is that ok, or I should add more description?

Thanks @peymanslh . for the context of this PR I think that's enough. but we definitely need a complete README for the project

**Review by hramezani — APPROVED** [review]

File: src/manage.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import os
 import sys
 
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (mirzakhany) [inline src/manage.py]: please pass `override=True` to make sure that `os.environ.get` will return same data as we have in `.env`.
Or refactor settings module to use dotenv library.

Comment (peymanslh) [reply src/manage.py]: Fixed

File: .sample_env [hunk 2]
```diff
@@ -0,0 +1,18 @@
+DEBUG=True
+TEMPLATE_DEBUG=True
+SQL_DEBUG=True
+CACHE_BACKEND=
+ENABLE_SENTRY=False
+SENTRY_REDIS=False
+SENTRY_CELERY=False
+SENTRY_DSN=
+ALLOWED_HOSTS=localhost
+POD_IP=127.0.0.1
```
Comment (saeedghx68) [inline .sample_env:10]: @mirzakhany Is it needed to have it here?

File: src/manage.py [hunk 3]
```diff
@@ -3,6 +3,10 @@
 import os
 import sys
 
+from dotenv import load_dotenv
+
+load_dotenv(override=True)
```
Comment (saeedghx68) [inline src/manage.py:8]: @peymanslh, I think we should move these lines to the settings.py 
@hramezani what do you think?

**Review by saeedghx68 — COMMENTED** [review]
@peymanslh please squash your commits

File: .sample_env (hunk 2 above)
Comment (peymanslh) [reply .sample_env:10]: I listed all the environment variables we use in settings
which one do you think is irrelevant?

Comment (saeedghx68) [reply .sample_env:10]: I mean the `POD_ID` env is needed or not? mohsen can help us.

Comment (peymanslh) [reply .sample_env:10]: We used it in `src/proj/settings/base.py` line `57`
@mirzakhany 
```python
ALLOWED_HOSTS.append(os.environ.get("POD_IP", ""))
```

File: src/manage.py (hunk 3 above)
Comment (hramezani) [reply src/manage.py:8]: I think we can keep it at `manage.py` as we can see something like that in [django-dotenv](https://github.com/jpadilla/django-dotenv).

**Review by saeedghx68 — APPROVED** [review]

**Pull request merged**